articles/README.md

```diff
@@ -41,6 +41,35 @@ for contributors:
 - The titles of the articles, the names in the TOC and the *uid* of the metadata
   should be as close as possible among them and represent clearly the H1 header
   of the Markdown document.
+  
+## Broken links prevention
+
+The url in the `docs.microsoft.com` site is generated from the path relative to `articles`on the GitHub repository. For example, the article located in `./articles/user-guide/programs.md` has assigned as url `https://docs.microsoft.com/azure/quantum/user-guide/programs`. This fact means that each time an article changes it's path, either because of a relocation or a change in the name of the file, the url will change accordingly. This change can cause some users to find broken links to previously existing articles. To avoid this:
+
+- Try to not rename or re-locate the articles.
+- If for some reason you need to rename, remove or re-locate an article, you should add a redirect from the previous article to the new url to avoid creating any broken link.
```

Comment:
```suggestion
- If for some reason you need to rename, remove or relocate an article, you should add a redirect from the previous article to the new URL to avoid creating any broken links.
```